subsys/bluetooth/mesh/friend.c

```diff
@@ -1278,6 +1278,10 @@ static void friend_lpn_enqueue_rx(struct bt_mesh_friend *frnd,
 	struct friend_pdu_info info;
 	struct net_buf *buf;
 
+	if (bt_mesh_elem_find(rx->ctx.addr)) {
```

Comment:
Maybe a code comment here? Essentially the same as you have in the commit message. Is this a bug fix btw, or something that became necessary with this PR?

Reply:
This is a bug fix that is necessary for this patch to work correctly.